src/main/java/io/split/android/client/storage/splits/SplitsStorageImpl.java

```diff
@@ -135,6 +143,26 @@ public void clear() {
         mInMemorySplits.clear();
         mChangeNumber = -1;
         mPersistentStorage.clear();
+        mFlagSets.clear();
+        mTrafficTypes.clear();
+    }
+
+    @NonNull
+    @Override
+    public Set<String> getNamesByFlagSets(List<String> sets) {
```

Comment:
Can we reuse logic in `LocalhostSplitsStorage::getNamesByFlagSets` and `SplitsStorageImpl::getNamesByFlagSets`? I guess it is not as trivial as it looks like anyway, because it might require an abstract class to extend LocalhostSplitsStorage and SplitsStorageImpl from.

Reply:
Yes, will do when refactoring Localhost